doc/source/user/numpy-for-matlab-users.rst

```diff
@@ -258,63 +187,63 @@ Linear Algebra Equivalents
      - Notes
 
    * - ``ndims(a)``
-     - ``ndim(a)`` or ``a.ndim``
-     - get the number of dimensions of an array
+     - ``np.ndim(a)`` or ``a.ndim``
+     - get the number of dimensions of the array, ``a``
 
    * - ``numel(a)``
-     - ``size(a)`` or ``a.size``
-     - get the number of elements of an array
+     - ``np.size(a)`` or ``a.size``
+     - get the number of elements of the array, ``a``
 
    * - ``size(a)``
-     - ``shape(a)`` or ``a.shape``
-     - get the "size" of the matrix
+     - ``np.shape(a)`` or ``a.shape``
+     - get the "size" of the array, ``a``
 
    * - ``size(a,n)``
      - ``a.shape[n-1]``
      - get the number of elements of the n-th dimension of array ``a``. (Note
-       that MATLAB® uses 1 based indexing while Python uses 0 based indexing,
+       that MATLAB uses 1 based indexing while Python uses 0 based indexing,
        See note :ref:`INDEXING <numpy-for-matlab-users.notes>`)
 
    * - ``[ 1 2 3; 4 5 6 ]``
-     - ``array([[1.,2.,3.], [4.,5.,6.]])``
-     - 2x3 matrix literal
+     - ``np.array([[1.,2.,3.], [4.,5.,6.]])``
+     - 2D array defined as 2x3 matrix
 
    * - ``[ a b; c d ]``
-     - ``block([[a,b], [c,d]])``
+     - ``np.block([[a,b], [c,d]])``
      - construct a matrix from blocks ``a``, ``b``, ``c``, and ``d``
 
    * - ``a(end)``
      - ``a[-1]``
-     - access last element in the 1xn matrix ``a``
+     - access last element in MATLAB vector (1xn or nx1) or 1D NumPy array
+       ``a`` (length n)
 
    * - ``a(2,5)``
      - ``a[1,4]``
-     - access element in second row, fifth column
+     - access element in second row, fifth column in a 2D array, ``a``
 
    * - ``a(2,:)``
      - ``a[1]`` or  ``a[1,:]``
-     - entire second row of ``a``
+     - entire second row of a 2D array, ``a``
 
    * - ``a(1:5,:)``
      - ``a[0:5]`` or ``a[:5]`` or ``a[0:5,:]``
-     - the first five rows of ``a``
+     - the first five rows of a 2D array, ``a``
 
    * - ``a(end-4:end,:)``
      - ``a[-5:]``
-     - the last five rows of ``a``
+     - the last five rows of a 2D array, ``a``
 
    * - ``a(1:3,5:9)``
-     - ``a[0:3][:,4:9]``
-     - rows one to three and columns five to nine of ``a``.  This gives
-       read-only access.
+     - ``a[0:3,4:9]``
+     - rows one to three and columns five to nine of a 2D array, ``a``. 
```

Comment:
We can't say "row one" if the two notations disagree on what row 1 is. We need to stick to ordinals or draw a picture.

Reply:
@bjnath, do you prefer this?
```suggestion
     - The first through third rows and fifth through ninth columns of a 2D array, ``a``. 
```

Reply:
@eric-wieser Yes, this is what I had in mind.